### PR TITLE
fix(vs): Ensure that infobars get closed properly

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: src/Uno.UI.RemoteControl.VS/Commands/UnoMenuCommand.cs [hunk 1]
```diff
@@ -124,10 +136,21 @@ private void OnBeforeQueryStatusDynamicItem(object sender, EventArgs args)
...
+	private static int GetCurrentPosition(DynamicItemMenuCommand matchedCommand)
+		// The position of the command is the command ID minus the ID of the root dynamic start item.
+		=> matchedCommand.MatchedCommandId == 0 ? 0 : matchedCommand.MatchedCommandId - (int)DynamicMenuCommandId;
 
 	private bool TryGetCommandRequestIdeMessage(DynamicItemMenuCommand matchedCommand, [NotNullWhen(true)] out AddMenuItemRequestIdeMessage result)
 		=> (result = CommandList.Skip(GetCurrentPosition(matchedCommand)).FirstOrDefault()) != null;
+
+	public void Dispose()
+	{
+		CommandService.RemoveCommand(_dynamicMenuCommand);
```
Comment (ajpinedam) [inline src/Uno.UI.RemoteControl.VS/Commands/UnoMenuCommand.cs:148]: Is it ok passing null into here? (could not find `RemoveCommand` implementation to validate it myself

**Review by ajpinedam — APPROVED** [review]

File: src/Uno.UI.RemoteControl.VS/Commands/UnoMenuCommand.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI.RemoteControl.VS/Commands/UnoMenuCommand.cs:148]: Fair! Those apis are not nullable enable, it does not show in compiler.

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.5

**Pull request merged**